.gitignore

```diff
@@ -63,3 +63,5 @@ dump.rdb
 /.project
 .generators
 .rakeTasks
+
+.talismanrc
```

Comment:
I'm not certain we're using Talisman in our git hooks for this proejct, can this be added to your local `~/.gitignore.global` ?

Reply:
Good spot, we've just removed it! Thanks.